src/transformers/tf_utils.py

```diff
@@ -22,6 +23,8 @@
 
 logger = logging.get_logger(__name__)
 
+build_context = threading.local()
```

Comment:
This is not used anywhere here?

Reply:
Ah, you're right, sorry! This is leftover from an earlier approach I was trying.

Reply:
Fixed!